### DISCUSSION TITLE
Is there a way to quickly navigate map library in lazer?

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#22468] [Q&A] [answered]
I don't know if it's intended behavior but right-clicking a bit left to bottom/top of the song list doesn't move you to that position in the song list as it does in stable. Is there any other way to quickly navigate through songs?

**Comment by NotGumballer91** [comment, marked as answer]
Enable this

![image](https://user-images.githubusercontent.com/64581009/215777221-c114e208-d29f-4585-b7cd-04eab30199b1.png)

**Reply by crumpledmint** [reply]
Thanks! Couldn't find it in the settings by myself
